pkg/util/filepath.go

```diff
@@ -84,17 +83,22 @@ func walk(path string, info os.FileInfo, resolvedPath string, symlinkPathsFollow
 		}
 		return walk(path, info2, path2, symlinkPathsFollowed, walkFn)
 	} else if info.IsDir() {
-		list, err := ioutil.ReadDir(path)
+		list, err := os.ReadDir(path)
 		if err != nil {
 			return walkFn(resolvedPath, info, err)
 		}
 		var subFiles = make([]subFile, 0)
-		for _, fileInfo := range list {
-			path2 := filepath.Join(path, fileInfo.Name())
+		for _, file := range list {
+			path2 := filepath.Join(path, file.Name())
 			var resolvedPath2 string
 			if resolvedPath != "" {
-				resolvedPath2 = filepath.Join(resolvedPath, fileInfo.Name())
+				resolvedPath2 = filepath.Join(resolvedPath, file.Name())
 			}
+			fileInfo, err := file.Info()
+			if err != nil {
+				continue
+			}
+
```

Comment:
Should we add a log here? 

Previous behavior would be that ioutil would just error and return

Reply:
Where is this used and would ioutil error if it cannot stat an individual file?

I'm not sure what behavior makes the most sense here, I guess that sort of depends on the caller's expectations.

I guess we could approach this as either of
1. Ignore quietly
2. Ignore and log
3. Return an error on stat failure
4. Append file without fileInfo and deal with it later

From a quick glance it seems like 2 might make the most sense, but it could go either way 🤷.